frontend/src/components/languages/index.tsx

```diff
@@ -133,21 +126,33 @@ export const Languages = () => {
     },
     {
       flex: 1,
-      field: "isDefault",
-      headerName: t("label.is_default"),
+      field: "isRTL",
+      headerName: t("label.is_rtl"),
       disableColumnMenu: true,
       renderHeader,
       headerAlign: "left",
       valueGetter: (value) => (value ? t("label.yes") : t("label.no")),
     },
     {
-      flex: 1,
-      field: "isRTL",
-      headerName: t("label.is_rtl"),
+      maxWidth: 120,
+      field: "isDefault",
+      headerName: t("label.is_default"),
       disableColumnMenu: true,
       renderHeader,
       headerAlign: "left",
-      valueGetter: (value) => (value ? t("label.yes") : t("label.no")),
+      renderCell: (params) => (
+        <Switch
+          checked={params.value}
+          color="primary"
+          inputProps={{ "aria-label": "primary checkbox" }}
+          disabled={
+            !hasPermission(EntityType.LANGUAGE, PermissionAction.UPDATE)
+          }
```

Comment:
```suggestion
          disabled={
            params.value ||
            !hasPermission(EntityType.LANGUAGE, PermissionAction.UPDATE)
          }
```
Hi @marrouchi, i think that this switch element can be disabled when it the value is not falsy

Reply:
Yes indeed 🙏